qemu/tests/block_resize.py

```diff
@@ -73,7 +73,12 @@ def compare_block_size(session, block_cmd, block_pattern):
 
     drive_path = ""
     if params.get("os_type") == 'linux':
-        drive_id = params["blk_extra_params_%s" % data_image].split("=")[1]
+        pattern = r"(serial|wwn)=(\w+)"
+        match = re.search(pattern, params["blk_extra_params_%s" % data_image], re.M)
```

Comment:
line too long 84 characters

Reply:
@hereischen , have updated, thanks for your reviewing.